js-exercises/least-common-multiple/leastCommonMultiple.js

```diff
@@ -0,0 +1,16 @@
+
+function leastCommonMultiple(...args) {
+  let largestNumber = Math.max(args[0],args[1])
+  let smallNumber = Math.min(args[0],args[1])
+  
+  let lcm = largestNumber
+  while(lcm%smallNumber != 0)
```

Comment:
What if one number is 0? It will create an infinite loop, so you need to handle that case as well.
`if(small == 0) return large`